app/src/main/java/com/example/concordia_campus_guide/Models/Building.java

```diff
@@ -0,0 +1,153 @@
+package com.example.concordia_campus_guide.Models;
+
+import android.text.TextUtils;
+
+import java.util.List;
+
+public class Building {
```

Comment:
Can all the info that will be displayed on the info card be wrapped as BuildingInfo? Just for the sake of making this more concise.

Reply:
Thomas, Tarek and I decided on merging everything into one Building class actually.